bin/compile

```diff
@@ -141,6 +141,11 @@ create_build_env
 [ ! "$NPM_CONFIG_CACHE" ] && NPM_CONFIG_CACHE=$(mktemp -d -t npmcache.XXXXX)
 export YARN_CACHE_FOLDER NPM_CONFIG_CACHE
 
+if [[ $(features_get "cache-native-yarn-cache") == "false" ]]; then
+  warn "'cache-native-yarn-cache' flag no longer used. Set an environment variable named YARN_CACHE to false to opt out of yarn caching."
```

Comment:
Can we change this to say "You're currently using the heroku-buildpack-features file to configure. Use 'heroku config:set YARN_CACHE=false' to set the variable. Using Yarn Cache has moved behind an environment variable."